.travis.yml

```diff
@@ -39,6 +39,9 @@ before_install:
     - edm install -y --version 3.6 click setuptools
     - git clone git://github.com/force-h2020/force-bdss.git
     - pushd force-bdss && edm run -- python -m ci build-env && edm run -- python -m ci install && popd
+    - git clone git://github.com/force-h2020/force-wfmanager.git
+    - pushd force-wfmanager && edm run -- python -m ci install && popd
+    - if [[ ${TRAVIS_OS_NAME} == "linux" ]] ; then sudo apt-get install --yes libglu1-mesa-dev mesa-common-dev; fi
```

Comment:
Is this required by the WorkflowManager?

Reply:
The CI requires WfManager to run fully, so we need to install it on Travis even though the plugin can be installed locally without WfManager.

The final line is needed by the UI only if running on Travis: https://github.com/force-h2020/force-wfmanager/pull/284

Reply:
Right, thank you. I forgot that we have a contributed UI module here.